projectgenerator/utils/qt_utils.py

```diff
@@ -81,6 +81,21 @@ def selectFolder(line_edit_widget, title, parent):
 def make_folder_selector(widget, title=QCoreApplication.translate('projectgenerator', 'Open Folder'), parent=None):
     return partial(selectFolder, line_edit_widget=widget, title=title, parent=parent)
 
+def color_log_text(text, txt_edit):
+    textlines = text.splitlines()
+    for textline in textlines:
+        if textline.startswith("Info:"):
```

Comment:
Can we have this code somewhere in `libili2db`? we are actually parsing ili2db output here, it's not that much related to qt.